src/bundle/Resources/config/routing.yml

```diff
@@ -8,3 +8,8 @@ ez_hybrid_platform_ui_version_draft_actions:
     defaults:
         _controller: EzSystemsHybridPlatformUiBundle:Version:draftActions
     methods: [POST]
+ez_hybrid_platform_ui_location_actions:
+    path: /location/actions/{contentId}/{locationId}
```

Comment:
The `locationId` parameter should be passed as a query param: `/location/actions/{contentId}?redirectLocationId={locationId}`

The action should redirect to that ID if given, or to the main if not given.